pymc_experimental/model_transform/conditioning.py

```diff
@@ -206,3 +213,132 @@ def do(
     if prune_vars:
         return prune_vars_detached_from_observed(model)
     return model
+
+
+def change_value_transforms(
+    model: Model,
+    vars_to_transforms: Dict[ModelVariable, Union[RVTransform, None]],
+) -> Model:
+    """Change the value variables transforms in the model
+
+    Parameters
+    ----------
+    model: Model
+    vars_to_transforms: Dict
+        Mapping between RVs and new transforms to be applied to the respective value variables
```

Comment:
```suggestion
    vars_to_transforms: Mapping[ModelVariable, Union[RVTransform, None]],
) -> Model:
    """Change the value variables transforms in the model

    Parameters
    ----------
    model : Model
    vars_to_transforms : dict-like
        Mapping from RVs to new transforms to be applied to the respective value variables
```

Reply:
Took some suggestions, not sure about changing `Dict` to `Mapping`

Reply:
Usually mypy complains about using `Dict` for a parameter

Reply:
> Usually mypy complains about using `Dict` for a parameter

Do you have some link? I never faced that

Reply:
https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance

Reply:
Changed